### PR TITLE
Add change location screen

### PR DESCRIPTION
**Pull request by saugatdhimal**

### CODE REVIEW

File: src/components/Map/styles.tsx [hunk 1]
```diff
@@ -32,4 +32,29 @@ export default StyleSheet.create({
...
+        width: 2,
+        backgroundColor: '#00518B',
+    },
+    markerDotOuter: {
+        width: 25,
+        height: 25,
+        borderRadius: 25,
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: 'white',
```
Comment (PDipendra) [inline src/components/Map/styles.tsx]: colors.white

File: src/components/Map/styles.tsx [hunk 2]
```diff
@@ -32,4 +32,29 @@ export default StyleSheet.create({
...
+    },
+    markerDotInner: {
+        width: 14,
+        height: 14,
+        borderRadius: 14,
+        backgroundColor: '#00518B',
```
Comment (PDipendra) [inline src/components/Map/styles.tsx]: blueTextAlt

File: src/components/Map/styles.tsx [hunk 3]
```diff
@@ -32,4 +32,29 @@ export default StyleSheet.create({
...
+
+    markerContainer: {
+        alignItems: 'center',
+        width: 45,
+        backgroundColor: 'transparent',
+    },
+    markerLine: {
+        height: 12,
+        width: 2,
+        backgroundColor: '#00518B',
```
Comment (PDipendra) [inline src/components/Map/styles.tsx]: blueTextAlt

File: src/screens/ChangeLocation/index.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    const [selectedMethod, setSelectedMethod] = useState('');
+    const handleSelectedMethod = useCallback((label: string) => {
+        setSelectedMethod(label);
+    }, []);
+
+    return (
+        <SafeAreaView style={styles.container}>
+            <View>
+                <InputField searchInput placeholder="ABC County" />
+                {Labels.map(label => {
```
Comment (PDipendra) [inline src/screens/ChangeLocation/index.tsx]: use flatlist

**Review by PDipendra — COMMENTED** [review]
navigate change location screen

File: src/screens/ChangeLocation/index.tsx (hunk 4 above)
Comment (saugatdhimal) [reply src/screens/ChangeLocation/index.tsx]: thanks

**Review by PDipendra — APPROVED** [review]

**Pull request merged**